src/Microsoft.AspNet.Identity/RoleManager.cs

```diff
@@ -60,6 +66,11 @@ public class RoleManager<TRole> : IDisposable where TRole : class
         protected IRoleStore<TRole> Store { get; private set; }
 
         /// <summary>
+        ///     Used to create scopes and messages for logging
+        /// </summary>
+        protected internal virtual IdentityLogger IdentityLogger { get; set; }
```

Comment:
I chose to keep it IdentityLogger since it contains the ILogger implementation property which is named `Logger` so a call to it would be `Logger.Logger` which now is `IdentityLogger.Logger`. I would have to rename the ILogger to something else but Logger seems to be a good choice for it. Let me know if you feel strongly about it